### DISCUSSION TITLE
Feature Request: Nintendont video width parameter

### DISCUSSION THREAD

**Discussion by supertazon** [Aurelio92/RVLoader#27] [Ideas]
It would be awesome to be able to set the "video width" Nintendont parameter straight from RVLoader. Mainly to be able to set it at 720 to get a fullscreen image using GCVideo on a 3.5 inch IPS screen. I am sure it would be useful in many other cases.

**Comment by toyojosealejandro-lang** [comment]
Agradecería de todo corazón que implementaran un wad/app de desinstalación definitiva del RVLoader en verdad, los pasos para su desinstalación son un poco ambiguos y por lo menos en mi caso no me funcionaron los pasos que promocionaron, muchas gracias por su atención  
---------------------------------------------------------------------------------------------------------------------------------------------------------
I would sincerely appreciate it if you could implement a wad/app for the complete uninstallation of RVLoader. Honestly, the steps for its removal are a bit unclear, and at least in my case, the steps that were promoted didn't work. Thank you very much for your attention.
